packages/cashier/src/pages/account-transfer/account-transfer-form/__tests__/account-transfer-form.spec.tsx

```diff
@@ -342,4 +342,176 @@ describe('<AccountTransferForm />', () => {
+                is_crypto: false,
+                platform_icon: 'IcDeriv X',
+                text: 'Deriv X',
+                value: 'DXR1029',
+            },
+            {
+                text: 'USD',
+                currency: 'USD',
+                value: 'MTR40013177',
+                platform_icon: 'IcMt5Derived',
+                is_crypto: false,
+                is_mt: true,
+                is_dxtrade: false,
+                is_derivez: false,
+            },
+        ];
+
+        // mockRootStore.modules.cashier.account_transfer.selected_to;
+        // mockRootStore.modules.cashier.account_transfer.selected_from;
+
+        const derivez_account = {
+            currency: 'USD',
+            is_mt: false,
+            is_dxtrade: false,
+            is_derivez: true,
+            is_crypto: false,
+            // platform_icon: "IcDerivez",
+            text: 'Deriv EZ',
+            value: 'EZR80000469',
+        };
+
+        const derivx_account = {
+            currency: 'USD',
+            is_mt: false,
+            is_dxtrade: true,
+            is_derivez: false,
+            is_crypto: false,
+            platform_icon: 'IcDxtradeDeriv X',
+            text: 'Deriv X',
+            value: 'DXR1029',
+        };
+
+        const currency_usd_account = {
+            text: 'USD',
+            value: 'CR90000212',
+            balance: '9953.89',
+            currency: 'USD',
+            is_crypto: false,
+            is_mt: false,
+            is_dxtrade: false,
+            is_derivez: false,
+        };
+
+        const currency_btc_account = {
+            text: 'BTC',
+            value: 'CR90000249',
+            currency: 'BTC',
+            is_crypto: true,
+            is_mt: false,
+            is_dxtrade: false,
+            is_derivez: false,
+        };
+
+        const mt5_account = {
+            text: 'USD',
+            currency: 'USD',
+            value: 'MTR40013177',
+            is_crypto: false,
+            is_mt: true,
+            is_dxtrade: false,
+            is_derivez: false,
+        };
+        // it('should show the deriv ez icon when deriv ez account is selected in the from or to account dropdown`', () => {
+
+        //     renderAccountTransferForm();
+        //     expect(screen.getByTestId('dt_IcDerivez')).toBeInTheDocument()
+
+        // });
+
+        describe('from_dropdown', () => {
+            it('should check for USD icon when USD is selected in from_dropdown', () => {
+                mockRootStore.modules.cashier.account_transfer.accounts_list = accountsList;
+                mockRootStore.modules.cashier.account_transfer.selected_from = currency_usd_account;
+                mockRootStore.modules.cashier.account_transfer.setTransferPercentageSelectorResult = jest
+                    .fn()
+                    .mockReturnValue(10.0);
+
+                renderAccountTransferForm();
+                expect(screen.getByTestId('dt-currency-usd')).toBeInTheDocument();
```

Comment:
Please follow the name convetion for the `testid`.

Link: [WikiJS](https://wikijs.deriv.cloud/en/Frontend/development-guidelines/test_guide#:~:text=Make%20sure%20you%20apply%20correct%20names%20while%20using%20data%2Dtestid%20attribute.%20It%20is%20required%20to%20use%20snake_case%20and%20dt%20prefix%3A)